html/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/style.css" />
+    <title>RaghadAbdulhadi Resume</title>
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <h1>RaghadAbdulhadi</h1>
+        </div>
+        <ul>
+          <li><a href="">Home</a></li>
+          <li><a href="">Education</a></li>
+          <li><a href="">Experience</a></li>
+          <li><a href="">Projects</a></li>
+        </ul>
+      </nav>
+      <div class="about-section">
```

Comment:
you are naming the class as about section while its a dive, its better to name it about-container or change the tag into section